store/src/java/com/zimbra/cs/store/BlobBuilder.java

```diff
@@ -235,6 +238,27 @@ public Blob finish() throws IOException, ServiceException {
         if (digest != null) {
             blob.setDigest(ByteUtil.encodeFSSafeBase64(digest.digest()));
             blob.setRawSize(totalBytes);
+            File file = blob.getFile();
+            File uncompresedFile = null;
+            if (blob.isCompressed() && totalBytes == file.length())
+            {
+              ZimbraLog.store.info("Blob compression is useless avoid it");
```

Comment:
Make this debug level - how about:
`ZimbraLog.store.debug("Blob compression is useless avoid it for %s size=%s", file, totalBytes)`